lib/Staticman.js

```diff
@@ -46,7 +46,7 @@ Staticman.prototype._applyTransforms = function (data) {
 }
 
 Staticman.prototype._checkForSpam = function (fields) {
-  if (!this.config.akismet || !this.config.akismet.enabled) return Promise.resolve(fields)
+  if (!this.config.akismet || !this.config.akismet.type) return Promise.resolve(fields)
```

Comment:
(Sorry, finally having enough time to properly look into this!)

Why did you change this property?